web-services/security/src/main/java/datawave/security/cache/AccumuloCacheStore.java

```diff
@@ -261,8 +260,7 @@ public boolean contains(Object key) {
     
     @Override
     public int size() {
-        try {
-            BatchScanner batchScanner = connector.createBatchScanner(tableName, authorizations, 5);
+        try (BatchScanner batchScanner = connector.createBatchScanner(tableName, authorizations, 5)) {
             batchScanner.setRanges(Collections.singleton(new Range()));
             try {
                 int rows = 0;
```

Comment:
can remove `batchScanner.close()` on line 272

Reply:
Fixed in next commit

Reply:
Another one missed... The close is still on line 272.